spec/unit/provider/service/systemd_spec.rb

```diff
@@ -134,6 +171,17 @@
       provider.disable
     end
   end
+  
+  describe "#mask" do
+    it "should run systemctl to disable and mask a service" do
+      provider = described_class.new(Puppet::Type.type(:service).new(:name => 'sshd.service'))
+      # I think that this is a symbol method due to the direct call from inside
+      # the provider. I'm not 100% sure.
+      provider.expects(:systemctl).with(:disable, 'sshd.service')
```

Comment:
:disable is an outlier in the provider. I'm not changing it due to potential other ramifications that have nothing to do with my patch.